README.md

```diff
@@ -29,6 +29,7 @@
 - [ ] String Translator
 - [ ] Asset Studio (Drawable & Icon Maker)
 - [x] Git (*)
+- [ ] Plugins

```

Comment:
You're not supposed to add these yourself.

Reply:
OK